Brightness Sync/AppDelegate.swift

```diff
@@ -139,61 +159,95 @@ class AppDelegate: NSObject, NSApplicationDelegate {
                 os_log("Display without en_US name found.")
             }
         }
-        
+
         return ultraFineDisplays
     }
-    
+
     static func getDisplayInfoDictionaries() -> [NSDictionary] {
         var diplayInfoDictionaries = [NSDictionary]()
-        
+
         var iterator: io_iterator_t = 0
         guard IOServiceGetMatchingServices(kIOMasterPortDefault, IOServiceMatching("IODisplayConnect"), &iterator) == 0 else {
             return diplayInfoDictionaries
         }
-        
+
         var display = IOIteratorNext(iterator)
-        
+
         while display != 0 {
             if let displayInfo = IODisplayCreateInfoDictionary(display, 0)?.takeRetainedValue() as NSDictionary? {
                 diplayInfoDictionaries.append(displayInfo)
             }
-            
+
             IOObjectRelease(display)
-            
+
             display = IOIteratorNext(iterator)
         }
-        
+
         IOObjectRelease(iterator)
-        
+
         return diplayInfoDictionaries
     }
-    
+
     @objc func copyDiagnosticsToPasteboard() {
         let CGDisplays = AppDelegate.getAllDisplays()
         let IODisplays = AppDelegate.getDisplayInfoDictionaries()
-        
+
         let diagnostics = """
         CGDisplayList:
         \(CGDisplays.map {
         ["VendorNumber": CGDisplayVendorNumber($0),
         "ModelNumber": CGDisplayModelNumber($0),
         "SerialNumber": CGDisplaySerialNumber($0)]
         })
-        
+
         IODisplayList:
         \(IODisplays)
         """
-        
+
         NSPasteboard.general.clearContents()
         NSPasteboard.general.setString(diagnostics, forType: .string)
     }
-    
+
     @objc func checkForUpdates() {
         NSWorkspace.shared.open(URL(string: "https://github.com/OCJvanDijk/Brightness-Sync/releases")!)
     }
-    
+
+    @objc func brightnessOffsetUpdated(slider: NSSlider) {
+        brightnessOffset = slider.doubleValue
+        syncTimer?.fire()
+    }
+
     struct DisplayIdentifier: Hashable {
         let vendorNumber: UInt32
         let modelNumber: UInt32
     }
+
+    private lazy var sliderView: NSView = {
+        let container = NSView(frame: NSRect(origin: CGPoint.zero, size: CGSize(width: 160, height: 28)))
+
+        let updateSel = #selector(brightnessOffsetUpdated(slider:))
+        let statusSlider = NSSlider(value: brightnessOffset, minValue: -0.5, maxValue: 0.5, target: self, action: updateSel)
+        container.addSubview(statusSlider)
+        statusSlider.translatesAutoresizingMaskIntoConstraints = false
+
+        statusSlider.leadingAnchor.constraint(equalTo: container.leadingAnchor, constant: 20).isActive = true
```

Comment:
Is there a better way to match up the slider to the text, than using a magic number (`20`) here?

Reply:
I searched for it, but can't find anything. I made a screenshot of the system menu Volume slider and counted the pixels, so I'm going to just copy those measurements 😬